examples/funloc/funloc_params.yml

```diff
@@ -0,0 +1,189 @@
+paths:
+  subjects: ['subj_01', 'subj_02']
+  structurals: ['AKCLEE_107_slim', 'AKCLEE_110_slim']
+  dates: [[2014, 2, 14], null]
+  work_dir: '.'
+  subjects_dir: null
+  subject_run_indices: [null, [0]]
+  subject_indices: [0, 1]
+  disp_files: True
+  fetch_raw:
+    acq_ssh: minea
+    acq_dir:
+      - /sinuhe_data01/eric_non_space
+      - /data101/eric_non_space
+      - /sinuhe/data01/eric_non_space
+      - /sinuhe/data02/eric_non_space
+      - /sinuhe/data03/eric_non_space
+    acq_port: 22
+  scoring:
+    list_dir: lists
+    on_process: null
+  bads:
+    bad_dir: bads
+    bad_tag: _post-sss.txt
+  raw:
+    raw_dir: raw_fif
+    keep_orig: False
+    raw_fif_tag: _raw.fif
+    sss_fif_tag: _raw_sss.fif
+    run_names: ['%s_funloc']
+    runs_empty: ['%s_erm']
+  preprocessing:
+    sss_dir: sss_fif
+    pca_dir: sss_pca_fif
+  epochs:
+    epochs_dir: epochs
+    epochs_prefix: All
+    epochs_tag: -epo
+    eq_tag: eq
+  covariance:
+    cov_dir: 'covariance'
+  forward:
+    forward_dir: forward
+    trans_dir: trans
+  inverse:
+    inverse_dir: inverse
+    inv_tag: -sss
+    inv_erm_tag: -erm
+    inv_fixed_tag: -fixed
+    inv_loose_tag: ''
+    inv_free_tag: -free
+    inv_names: ['%s']
+    inv_runs: [0]
+
+scoring:
+  score: null
+
+preprocessing:
+  multithreading:
+    n_jobs: 1
+    n_jobs_mkl: 1
+    n_jobs_fir: 1
+    n_jobs_resample: 1
+  bads:
+    mf_autobad: True
+    mf_badlimit: 7
+    auto_bad: null
+    auto_bad_reject: null
+    auto_bad_flat: null
+    auto_bad_eeg_thresh: 10
+    auto_bad_meg_thresh: 10
+  head_position_estimation:
```

Comment:
Since this file will be used as a template by users of varying levels of background knowledge, I think it needs more scaffolding than it currently has.  It's easy enough to add in comments later, but we have the potential to make the file somewhat self-documenting by making the key names very clear.  WDYT?  This key `head_position_estimation` seems like a good example of that; can we do similarly for all keys, or would that be too much of a pain?

(note: even if we do spell out the keys clearly, some comments are probably warranted, e.g., for things like `n_jobs` we should say `# integer or "cuda"`)

Reply:
It would be nice if there were a way to only have to document in one place (here in the YAML?) and not also in the docstring of `Params`.  But if we do have to document in two places, I guess we could make the documentation in the YAML more sparse (i.e., limited just to the allowed values?)

Reply:
> can we do similarly for all keys, or would that be too much of a pain?

The main category names (epoching, etc.) we can change. The other ones would be a pain to change (esp. in a backward compatible way) because they map onto the `Params` class attributes.

Reply:
> The other ones would be a pain to change

OK, that's kinda what I thought